### PR TITLE
[minor] Support April Catalog Update

### PR DESCRIPTION
**Pull request by racree**
Ansible devops scripts updates to support April catalog 230411

See also:
- https://github.com/ibm-mas/cli/pull/236

### CODE REVIEW

File: docs/changes.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Changes
 Note that links to pull requests prior to public release of the code (4.0) direct to IBM GitHub Enterprise, and will only be accessible to IBM employees.
 
+- `13.2` Support April Catalog Update ([#759](https://github.com/ibm-mas/ansible-devops/pull/759))
```
Comment (terenceq) [inline docs/changes.md:4]: The PR is 762

File: ibm/mas_devops/README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 ## Change Log
 Note that links to pull requests prior to public release of the code (4.0) direct to IBM GitHub Enterprise, and will only be accessible to IBM employees.
 
+- `13.2` Support April Catalog Update ([#759](https://github.com/ibm-mas/ansible-devops/pull/759))
```
Comment (terenceq) [inline ibm/mas_devops/README.md:9]: The PR is 762

**Review by terenceq — APPROVED** [review]
Just update the PR number

File: docs/changes.md (hunk 1 above)
Comment (sanju7216) [reply docs/changes.md]: done

File: ibm/mas_devops/README.md (hunk 2 above)
Comment (sanju7216) [reply ibm/mas_devops/README.md]: done

**Review by terenceq — DISMISSED** [review]

**Review by terenceq — APPROVED** [review]

**Pull request merged**